### ISSUE TITLE
Syncthing returning to default folder  

### ISSUE THREAD

**Issue by Isa7q** [cizia64/CrossMix-OS#223]
I don't know if it's a problem with my connection, but syncthing changes the TPS destination folders to the default, having to redo the entire path again when it happens.

**Comment by dremond71** [comment]
I ran into this too, and I fixed it.

1. Take out your crossmix sd-card from the TrimUI Smart Pro
2. Put the sd card into an sd-card reader 
3. Plug it into a USB hub on a desktop computer (mine is Ubuntu)
4. On my Ubuntu computer, the location of the sd card is /media/my-user-name/CROSSMIX
5. Open a terminal window and go to that directory
6. cd './Apps/SystemTools/Menu/NETWORK##Syncthing (state)$'
7. In step 6, you need to surround the path with single quotes, since there are spaces.
8. The problematic file is: 'Syncthing Server  - enable.sh'
9. Open this file in an editor ( I used VSCode ).
10. Change

```sh
if ! [ -f /mnt/SDCARD/System/bin/syncthing/config.xml ]; then
```

to

```sh
if ! [ -f /mnt/SDCARD/System/etc/syncthing/config.xml ]; then
```
11. Save the file.
12. Close your editor
13. Close your terminal window
14. Eject your microSD card
15. Put it back into your TrimUI Smart Pro
16. When you enable or disable Syncthing, or turn your device on or off, your
      `/media/dremond/CROSSMIX-OS/System/etc/syncthing/config.xml` will return
      the info you set up, and will not default to the useless '/mnt/SDCARD/syncthing'
      value again.

**Comment by cizia64** [comment]
Hi it will be fixed in CrossMix v1.3.0 thanks to this update: https://github.com/cizia64/CrossMix-OS/commit/32825036c741647286d75eee801e595c3235c831

**Comment by cizia64** [comment]
I close, as the v1.3.0 is now released , feel free to open again if there are problems ;)
